string id
float64 pos
float64 vel
